ui/web-v2/Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN apk update \
 
 WORKDIR /etc/nginx
 
+COPY dist/apps/admin/ /var/www/
+
```

Comment:
I think this Dockerfile was for `bucketeer-web-nginx` originally.
By adding this command, I changed it for `bucketeer-web-v2`.

I used [this](https://github.com/bucketeer-io/bucketeer/blob/1c9cefbebb02c5f96e870b9223299b7e7c06e18a/ui/web-v2/BUILD.bazel#L25-L31) as reference.